dip-template/nodes/dip-consumer/src/service.rs

```diff
@@ -278,42 +297,12 @@ async fn start_node_impl(
 			transaction_pool,
 			sync_service.clone(),
 			params.keystore_container.keystore(),
-			force_authoring,
-			para_id,
-		)?;
-
-		let spawner = task_manager.spawn_handle();
-		let params = StartCollatorParams {
-			para_id,
-			block_status: client.clone(),
-			announce_block,
-			client: client.clone(),
-			task_manager: &mut task_manager,
-			relay_chain_interface,
-			sync_service: sync_service.clone(),
-			spawner,
-			parachain_consensus,
-			import_queue: import_queue_service,
-			collator_key: collator_key.expect("Command line arguments do not allow this. qed"),
 			relay_chain_slot_duration,
-			recovery_handle: Box::new(overseer_handle),
-		};
-
-		start_collator(params).await?;
-	} else {
-		let params = StartFullNodeParams {
-			client: client.clone(),
-			announce_block,
-			task_manager: &mut task_manager,
 			para_id,
-			relay_chain_interface,
-			sync_service,
-			relay_chain_slot_duration,
-			import_queue: import_queue_service,
-			recovery_handle: Box::new(overseer_handle),
-		};
-
-		start_full_node(params)?;
+			collator_key.expect("Command line arguments do not allow this. qed"),
```

Comment:
A better error message, like `"Collator key not provided"`?

Reply:
upd in: [e50e5d2](https://github.com/KILTprotocol/kilt-node/pull/627/commits/e50e5d2e7a90c5f16adc174bb65b638b2f7d678a)